kubedifflib/_diff.py

```diff
@@ -1,7 +1,9 @@
-from __future__ import print_function
-from __future__ import division
+#!/usr/bin/env python
```

Comment:
Is the shebang line needed in `kubedifflib`?

Reply:
It is not needed, but It is a common pattern to write modules for use as both a component and a main program. I'm going to remove in this context and added in the future if needed.